src/commands/set/index.ts

```diff
@@ -0,0 +1,8 @@
+import { BaseCommand } from '../base-command.ts';
+import SetSecretAccountCommand from './secret-account.ts';
+
+const SetCommands = BaseCommand().description('Set configuration options for Arcctl');
+
+SetCommands.command('secretAccount', SetSecretAccountCommand.alias('account'));
```

Comment:
Is it possible to use dot-notation in command names? I'm hoping that setting config fields like this can look like the following:

```sh
$ arcctl set state.backend ...
$ arcctl set defaults.registry ...
```

Also, can we call this field state backend instead of secret account?